easybuild/easyconfigs/s/scikit-learn/scikit-learn-0.20.3-foss-2019a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'scikit-learn'
+version = '0.20.3'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['c503802a81de18b8b4d40d069f5e363795ee44b1605f38bc104160ca3bfe2c41']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [('SciPy-bundle', '2019.03')]
+
+download_dep_fail = True
+use_pip = True
+
+# required because we're building a Python package using Intel compilers on top of Python built with GCC
+check_ldshared = True
```

Comment:
Technically speaking this is not correct in this case, at least the comment isn't.

Reply:
Yeah, it's not needed at all here, will remove, thanks.